src/microbe_stage/Microbe.cs

```diff
@@ -321,6 +321,11 @@ public float Radius
     /// </summary>
     public bool CanBind => organelles.Any(p => p.IsBindingAgent) || Colony != null;
 
+    /// <summary>
+    ///   Returns true when this microbe has toxin vacuole and can fire toxins.
```

Comment:
toxin vacuoles

Reply:
actually, this `and can fire toxins` part is wrong. This also returns true, even if the player does not have any oxytoxy and therefore couldn't fire

Reply:
Wait, I just noticed the existence of `AgentVacuoleCount` property, I'll remove this in favor of using that one.